ConfigSpace/hyperparameters/constant.pyx

```diff
@@ -0,0 +1,179 @@
+from typing import Any, Dict, List, Optional, Union
+
+import numpy as np
+cimport numpy as np
+np.import_array()
+
+# We now need to fix a datatype for our arrays. I've used the variable
+# DTYPE for this, which is assigned to the usual NumPy runtime
+# type info object.
+DTYPE = float
+# "ctypedef" assigns a corresponding compile-time type to DTYPE_t. For
+# every type in the numpy module there's a corresponding compile-time
+# type with a _t-suffix.
+ctypedef np.float_t DTYPE_t
+
+from ConfigSpace.hyperparameters.hyperparameter cimport Hyperparameter
+from ConfigSpace.hyperparameters.hyperparameter import Hyperparameter
```

Comment:
Why this `cimport` and regular `import`?

Reply:
I scrolled through the file briefly and only see `Hyperparameter` listed as the superclass of `Constant`. Seeing as it's a `cdef class`, I imagine we only need the `cimport` variant.

Reply:
Good catch, I'll remove the extra import.